### DISCUSSION TITLE
Multiarch

### DISCUSSION THREAD

**Discussion by slamd64** [xelalexv/dregsy#42] [Q&A]
Hello!

Is it possible to add multiarch docker sync to dregsy?
ARM64 architecture becomes very popular and AWS EKS/ECR already supports this. So, syncing AMD64+ARM64 is good idea.

https://docs.docker.com/docker-for-mac/multi-arch/

**Comment by slamd64** [comment]
As I see, Skopeo is already supports this feature. So, how to bypass "--all" from Dregsy command line to Skopeo?

skopeo copy
options:
--all
If source-image refers to a list of images, instead of copying just the image which matches the current OS and architecture (subject to the use of the global --override-os, --override-arch and --override-variant options), attempt to copy all of the images in the list, and the list itself.

**Comment by xelalexv** [comment]
Currently that's not supported, but would be a really good feature. Could you open an according issue?

**Reply by slamd64** [reply]
added https://github.com/xelalexv/dregsy/issues/43
